source/MapPanel.cpp

```diff
@@ -1016,7 +1016,7 @@ void MapPanel::DrawTravelPlan()
 	bool hasEscort = false;
 	map<const Ship *, double> fuel;
 	for(const shared_ptr<Ship> &it : player.Ships())
-		if(!it->IsParked() && !it->CanBeCarried() && it->GetSystem() == flagship->GetSystem())
+		if(!it->IsParked() && (!it->CanBeCarried() || it.get() == flagship) && it->GetSystem() == flagship->GetSystem())
```

Comment:
> In theory, you could extend this idea to any fighter (not just the flagship) that's been modified for jump travel (and isn't able to be carried). But that would be a lot more work. ;)

Couldn't we do something like `!it->CanBeCarried() || it->JumpNavigation().HasTravel()` where `HasTravel()` is a new ShipJumpNavigation method that is shorthand for `HasHyperdrive() || HasJumpDrive()`?

Reply:
Kinda.  The thing is, fighters modified for jump travel can still be carried.  The only fighter that simply can't be carried is the flagship.  Other fighters _might_ be carried, depending on the fleet composition, which is a bit more complex.  And is it worth figuring all that out every Draw cycle for a case that almost never comes up?  (I know, you could say the same of this whole PR, but the modification I suggested was a pretty minor one.)

Reply:
`it->GetSystem() == flagship->GetSystem()` already filters out docked fighters, as `GetSystem()` returns nullptr for docked fighters. `!it->CanBeCarried()` is therefore only currently filtering deployed fighters that are in the same system. This actually means we could completely replace `!it->CanBeCarried()` with `it->JumpNavigation().HasTravelMethod()`, as this would still filter out deployed fighters that can't travel on their own, but would include deployed fighters which can travel on their own (which is what we're talking about wanting). This would actually also filter out non-fighters that have no travel method, which might be the case if you intentionally leave warships without hyperdrives laying around in a system.

Reply:
>it->GetSystem() == flagship->GetSystem() already filters out docked fighters, as GetSystem() returns nullptr for docked fighters. 

But when you're on a planet, the fighters aren't docked.  HasAnyJumpMethod will certainly filter out non-jump fighters, but you're still potentially left with fighters that have jump drives that will actually get carried.

Reply:
There's probably some slick way we could figure it out, but it probably is better to just always ignore fighters unless it's the flagship.